spigot/src/main/java/com/github/syldium/nethertree/runnable/RunnablesManager.java

```diff
@@ -119,6 +119,21 @@ public void save() {
         });
     }
 
+    public void saveCancelAndClear() {
+        getFileConfiguration(false).ifPresent(config -> {
+            for (Map.Entry<UUID, DecayRunnable> entry : this.runnables.entrySet()) {
+                config.set(entry.getKey().toString(), entry.getValue().serialize());
+                entry.getValue().cancel();
+            }
+            this.runnables.clear();
+            try {
+                config.save(this.fileConfig);
+            } catch (IOException e) {
+                e.printStackTrace();
+            }
+        });
```

Comment:
```suggestion
        save();
        this.runnables.clear();
```
I think the `RunnablesManager` class may just be re-instantiated in case you want to reload.

Reply:
lol, oops, yep